app/assets/stylesheets/layout/_footer.scss

```diff
@@ -1 +1,44 @@
-// Your footer style
+.footer-container{
+  background-color: #31081F;
```

Comment:
lets use variables name like $brand-color or whatever name variable you want to give